app/parsers/bulkrax/csv_parser.rb

```diff
@@ -192,12 +192,12 @@ def current_record_ids
 
       case importerexporter.export_from
       when 'all'
-        @work_ids = ActiveFedora::SolrService.query("has_model_ssim:(#{Hyrax.config.curation_concerns.join(' OR ')}) #{extra_filters}", method: :post, rows: 2_147_483_647).map(&:id)
+        @work_ids = ActiveFedora::SolrService.query("has_model_ssim:(#{Bulkrax.curation_concerns.join(' OR ')}) #{extra_filters}", method: :post, rows: 2_147_483_647).map(&:id)
         @collection_ids = ActiveFedora::SolrService.query("has_model_ssim:Collection #{extra_filters}", method: :post, rows: 2_147_483_647).map(&:id)
-        @file_set_ids = ActiveFedora::SolrService.query("has_model_ssim:FileSet #{extra_filters}", method: :post, rows: 2_147_483_647).map(&:id)
+        @file_set_ids = ActiveFedora::SolrService.query("has_model_ssim:#{Bulkrax.file_model_name.to_s} #{extra_filters}", method: :post, rows: 2_147_483_647).map(&:id)
```

Comment:
```suggestion
        @file_set_ids = ActiveFedora::SolrService.query("has_model_ssim:#{Bulkrax.file_model_name} #{extra_filters}", method: :post, rows: 2_147_483_647).map(&:id)
```

Reply:
since `value.to_s` and `"#{value}"` are equivalent.

Reply:
ooh yup!